content/downloading/contents.lr

```diff
@@ -14,6 +14,10 @@ The safest and simplest way to download Tor Browser is from the official Tor Pro
 
 However, there may be times when you cannot access the Tor Project website: for example, it could be blocked on your network. If this happens, you can use one of the alternative download methods listed below.
 
+##### Mirrors
+
+If you're unable to download Tor Browser from the official Tor Project website, you can instead try downloading it from one of our official mirrors, either through <a href="https://tor.eff.org">EFF</a> or <a href="http://tor.calyxinstitute.org">Calyx Institute</a>.
```

Comment:
"http://tor.calyxinstitute.org" always better to use https when available! ill correct it